src/main/java/org/isf/hospital/model/Hospital.java

```diff
@@ -122,8 +137,27 @@ public Hospital(String aCode, String aDescription, String aAddress,
         this.fax = aFax;
         this.email = aEmail;
         this.currencyCod = aCurrencyCod;
+		this.visitStartTime = Time.valueOf(VISIT_START_TIME);
+		this.visitEndTime = Time.valueOf(VISIT_END_TIME);
     }
-    
+
+	public Hospital(String aCode, String aDescription, String aAddress,
+			String aCity, String aTelephone, String aFax,
+			String aEmail, String aCurrencyCod,
+			Time visitStartTime, Time visitEndTime) {
+		super();
+		this.code = aCode;
+		this.description = aDescription;
+		this.address = aAddress;
+		this.city = aCity;
+		this.telephone = aTelephone;
+		this.fax = aFax;
+		this.email = aEmail;
+		this.currencyCod = aCurrencyCod;
+		this.visitStartTime = visitStartTime;
+		this.visitEndTime = visitEndTime;
+	}
+
     public String getAddress() {
         return this.address;
     }
```

Comment:
same here

Reply:
I reformatted the whole file to eliminate any formatting issues